tareas/clase-5/tareas-clase-5/tarea-4/tarea-4.js

```diff
@@ -0,0 +1,67 @@
+document.querySelector('#calcular-promedio').onclick = function () {
+    const numerosEnListaLi = document.querySelectorAll('.numero');
+
+    let listaDeNumeros = [];
+    for (let i = 0; i < numerosEnListaLi.length; i++) {
+        listaDeNumeros.push(Number(numerosEnListaLi[i].innerText));
+    }
+    function sumarNumeros(numeros) {
+        let sumaTotal = 0;
+        for (let i = 0; i < numeros.length; i++) {
+            sumaTotal = sumaTotal + numeros[i];
+        }
+        return sumaTotal;
+    }
+    let numeros = listaDeNumeros;
+    const sumaNumerosTotal = sumarNumeros(numeros);
+    document.querySelector(
+        '.suma-numeros-total',
+    ).innerText = `La suma total de todos los numeros es: ${sumaNumerosTotal}`;
+    
+    
+    function calcularPromedio(listaDeNumeros) {
+        let resultado = 0;
+        let promedio = 0;
+        for (let i = 0; i < listaDeNumeros.length; i++) {
+            resultado = resultado + listaDeNumeros[i];
+            promedio = resultado / listaDeNumeros.length;
+        }
+        return promedio;
+    }
+    document.querySelector('.numero-promedio').innerText = `El promedio es: ${calcularPromedio(listaDeNumeros)}`;
+    
+    function encontrarNumeroMinimo(listaDeNumeros) {
+        let numeroMinimo = listaDeNumeros[0];
+        for (let i = 0; i < listaDeNumeros.length; i++) {
+            if (listaDeNumeros[i] < numeroMinimo){
+                numeroMinimo = listaDeNumeros[i];
+            }
+            return numeroMinimo
```

Comment:
aunque tengas una lista de 1000000 números, después de chequear el primero ya hace el return, porque está dentro del for.